web/files/referencedby.go

```diff
@@ -83,5 +83,5 @@ func RemoveReferencedHandler(c echo.Context) error {
 		return wrapVfsError(err)
 	}
 
-	return c.NoContent(http.StatusNoContent)
+	return c.NoContent(http.StatusOK)
```

Comment:
Returning 204 causes Go to produce the following error: "http: request method or response status code does not allow body". This is because in order to remove references we pass them in the body of the DELETE request.

I can revert back but I felt that having one error less would be better (or am I the only one that has it?).